### ISSUE TITLE
Inaccessible Modrinth description

### ISSUE THREAD

**Issue by triphora** [Sweenus/SimplySwords#45]
Hi, I am one of Modrinth's administrators and the lead moderator.

The current description is very inaccessible to screen readers and to those who don't have images loaded. I would like to ask you to provide a transcription of the images in [the description](https://modrinth.com/mod/simply-swords).

This is a requirement per section 2.1 of [Modrinth's content rules](https://modrinth.com/legal/rules#general-expectations):
> Project descriptions must also be accessible. For the most part, this means that descriptions cannot mostly consist of text within images, and necessary information cannot be obscured. 

As such, if the changes are not made soon, restrictions may be put on the mod's page.

**Comment by Sweenus** [comment]
Hi there, thanks for reaching out to me!

I've gone ahead and added alt text to each of the banner images. The alt text matches or very closely reflects the text found within the images, and should be sufficient for screen readers.

Please let me know if any additional changes are required.

**Comment by triphora** [comment]
Looks good - thanks!
